pkg/webhook/inject/inject_test.go

```diff
@@ -24,17 +26,23 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	controllerCfg "github.com/chaos-mesh/chaos-mesh/pkg/config"
+	"github.com/chaos-mesh/chaos-mesh/pkg/metrics"
 	"github.com/chaos-mesh/chaos-mesh/pkg/webhook/config"
 )
 
 var _ = Describe("webhook inject", func() {
+	var i Injector
```

Comment:
please initialize this log with constructor, only declaring without initializing makes no sense

Reply:
maybe you need to use different `Injector`s for different testcases.